tutorials/chapter1.md

```diff
@@ -505,13 +553,17 @@ int main() {
     SimplePublisherService publisherService(provider.getPublisher<std::string>("example2", 0, nullptr, nullptr));
     SimpleSubscriberService subscriberService(provider.getSubscriber<std::string>("example2"));
 
-    publisherService.startup();
+    provider.start();
     subscriberService.startup();
+    publisherService.startup();
+    std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
Let's remove the `sleep_for` from these places. We can leave those inside loops to control the period, but those that are part of a init block should not exist.

Reply:
Done